pipeline/terraform/stack/cluster.tf

```diff
@@ -4,7 +4,7 @@ resource "aws_ecs_cluster" "cluster" {
 }
 
 module "inference_capacity_provider" {
-  source = "git::github.com/wellcomecollection/terraform-aws-ecs-service.git//modules/ec2_capacity_provider?ref=v3.4.0"
+  source = "git::github.com/wellcomecollection/terraform-aws-ecs-service.git//modules/ec2_capacity_provider?ref=v3.5.1"
```

Comment:
`1` not `2`?

Reply:
Oops, although not an issue for this module.